### PR TITLE
add roadmap for multi-cluster

### PR DESCRIPTION
**Pull request by yuswift**
Signed-off-by: yuswift <yuswiftli@yunify.com>

**What type of PR is this?**
 /kind documentation
**What this PR does / why we need it**:
Add roadmap docs for multi-cluster.

### CODE REVIEW

**Comment by zryfish** [comment]
Feature description is too short, I suggest adding more details.

**Comment by yuswift** [comment]
> Feature description is too short, I suggest adding more details.
In docs or issue page? I will add more details.

**Comment by yuswift** [comment]
@zryfish Updated the roadmap

**Comment by zryfish** [comment]
conflicts

**Comment by yuswift** [comment]
> conflicts

resolved

File: docs/roadmap.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
...
 
+### MultiCluster:
+
+- [x] Validate member cluster configuration if the member cluster config(e.g. jwtSecret) is not same as host cluster, which can reduce the complexity of joining clusters. [#3232](https://github.com/kubesphere/kubesphere/issues/3232)
+- [ ] Support configuring cluster controller resync period, some users need to update the cluster component more quickly. [#3213](https://github.com/kubesphere/kubesphere/issues/3213)
+- [x] Lightweight member cluster installation, support running kubepshere without redis and ldap and so on. [#3056](https://github.com/kubesphere/kubesphere/issues/3056)
+- [ ] The cluster controller updates the cluster.staus field instead of the tower server when using proxy connection.  [#3234](https://github.com/kubesphere/kubesphere/issues/3234)
+- [ ] Support  tower server and tower agent running highly available(server with leader election option and running more than one agent). [#31](https://github.com/kubesphere/tower/issues/31)
+- [ ] Support adding member clusters for an existing project(ns), wich can be helpful when there are new clusters joined.
```
[the remark this answers is not in the document]
Comment (yuswift) [reply docs/roadmap.md]: Done.

**Pull request merged**